### PR TITLE
Basic import for select in CSV

### PR DESCRIPTION
**Pull request by FelixMalfait**
Enables basic support for Select import and field matching in CSV. 
It's not pretty! But it's better than what we had before.
We should iterate on that quickly

Screenshot 2024-06-26 at 18 41 16

### CODE REVIEW

File: packages/twenty-front/src/modules/spreadsheet-import/utils/setColumn.ts [hunk 1]
```diff
@@ -12,33 +13,56 @@ export const setColumn = <T extends string>(
   field?: Field<T>,
   data?: MatchColumnsStepProps<T>['data'],
 ): Column<T> => {
-  switch (field?.fieldType.type) {
```
Comment (Weiko) [inline packages/twenty-front/src/modules/spreadsheet-import/utils/setColumn.ts:15]: Why removing the switch case pattern here? If it's too big maybe we can split the code inside functions?

**Review by Weiko — COMMENTED** [review]
Haven't tested the functionality yet but code LGTM! CI is failing on one of the tests 🙂 @FelixMalfait

File: packages/twenty-front/src/modules/spreadsheet-import/steps/components/ValidationStep/ValidationStep.tsx [hunk 2]
```diff
@@ -143,6 +143,10 @@ export const ValidationStep = <T extends string>({
               (importColumn) =>
                 (importColumn.type === ColumnType.matched &&
                   importColumn.value === column.key) ||
+                (importColumn.type === ColumnType.matchedSelect &&
```
[the remark this answers is not in the document]
Comment (FelixMalfait) [reply packages/twenty-front/src/modules/spreadsheet-import/steps/components/ValidationStep/ValidationStep.tsx:146]: Much nicer 😍 thanks

File: packages/twenty-front/src/modules/spreadsheet-import/utils/setColumn.ts (hunk 1 above)
Comment (FelixMalfait) [reply packages/twenty-front/src/modules/spreadsheet-import/utils/setColumn.ts:15]: Tbh I'm not sure why but the switch/case caused a wired typescript issue which I didn't figure out

File: packages/twenty-front/src/modules/spreadsheet-import/steps/components/ValidationStep/ValidationStep.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (FelixMalfait) [reply packages/twenty-front/src/modules/spreadsheet-import/steps/components/ValidationStep/ValidationStep.tsx:146]: Screenshot 2024-06-26 at 22 15 01

Not sure if I'm missing something but I feel like often when I try to do these kind of optimizations then Typescript gets lost

[the remark this answers is not in the document]
Comment (FelixMalfait) [reply packages/twenty-front/src/modules/spreadsheet-import/steps/components/ValidationStep/ValidationStep.tsx:146]: Screenshot 2024-06-26 at 22 17 42
Initially I had tried this but it was the same

**Pull request merged**